packages/app/components/Header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
```

Comment:
why do we need the "use client" here?

Reply:
You're importing a component that needs useState. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by default.